test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -213,6 +213,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.apm.featureFlags.migrationToFleetAvailable (any)',
         'xpack.apm.featureFlags.sourcemapApiAvailable (any)',
         'xpack.apm.featureFlags.storageExplorerAvailable (any)',
+        'xpack.apm.featureFlags.profilingIntegrationAvailable (any)',
```

Comment:
Can we specify 'boolean' here due to the strict schema?
https://github.com/elastic/kibana/blob/b8449e747de3ab3fdfba71bee35cc7057738382d/x-pack/plugins/apm/server/index.ts#L80

Reply:
Good point, done ✅